config/runtime.go

```diff
@@ -26,10 +26,25 @@ import (
 type Runtime struct {
 	DiagnosticsConfig DiagnosticsConfig         `yaml:"diagnostics,omitempty"`
 	HealthChecks      HealthChecksConfig        `yaml:"health-checks,omitempty"`
+	AuditConfig       *AuditConfig              `yaml:"audit,omitempty"`
```

Comment:
Do we expect any of this to actually be updated at runtime? Would it be simpler to put in install config?

Reply:
Similar to the above, this is based on the Java implementation. For Java products, the recommended way to adopt `audit.3` is to add the following block to the runtime configuration:

```
    audit:
      deployment: "{{environment.deployment}}"
      product-name: "{{conf.install.product-name}}"
      product-version: "{{conf.install.product-version}}"
      service: "{{service_name}}"
      stack: "{{stack_name}}"
      environment: "{{environment.name}}"
```

I'm not sure if a subset of these can be updated at runtime or if there's another reason for the block being there -- @IlanaRadinsky do you have any further context for this?